test/avocado/selenium-kdump.py

```diff
@@ -43,8 +42,12 @@ def requirements_cleanup(self):
             "echo '' | sudo tee {}".format(self.export_file),
             "sudo rm -rf {}".format(self.share),
         ]
+
         for exe in execs:
-            self.machine.execute(exe)
+            try:
+                self.machine.execute(exe)
+            except (RuntimeError, subprocess.CalledProcessError):
+                pass
```

Comment:
This pattern doesn't belong into a test, and is still bad even on cleanup -- there should not be such probabilistic/unpredictable commands and results.